instrumented/nightly-tests/src/main/kotlin/com/datadog/android/nightly/activities/ViewTrackingActivity.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.nightly.activities
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.datadog.android.nightly.R
+import com.datadog.android.nightly.utils.measure
+
+internal class ViewTrackingActivity : AppCompatActivity() {
+
+    val testMethodName = "rum_activity_view_tracking_strategy"
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.tracking_strategy_activity)
+    }
+
+    override fun onResume() {
+        measure(testMethodName) {
```

Comment:
why do we need this?

Reply:
I need to measure that what we do in the `onResume` of the activity lifecycle is under the hood is not affecting the perfs of this activity.

Reply:
Makes sense, then. Wasn't obvious from the beginning. Although we can have false positives here, because quite a lot of code in `onResume` is not owned by us.